packages/esm-appointments-app/src/header/appointments-header.component.tsx

```diff
@@ -19,6 +19,26 @@ const AppointmentsHeader: React.FC<AppointmentHeaderProps> = ({ title, appointme
   const { selectedDate, setSelectedDate } = useContext(SelectedDateContext);
   const { serviceTypes } = useAppointmentServices();
 
+  const items = [...serviceTypes];
```

Comment:
```suggestion
```

`items` seems unnecessary to me. We should be able to use `serviceTypes` directly.